src/test/resources/testing-editor.md

```diff
@@ -122,13 +124,18 @@ spec:
 ```
 -> push notification "create new"
 
+COMMENT: Clarify steps for creating new file: "Edit..." some resource (some file must be opened with editor) -> File -> New -> New File -> "foo.yml"
+
 **Pull notification "update existing" for new file with existing resource**
 1. make sure there's a task "foo" on cluster
 2. File > New > foo.yml
 3. paste yaml in previous use case
 
 -> Pull notification with "Push" link
 
+NOTE: foo.yml is already exists in /tmp. Cannot create file with same name.
```

Comment:
this is not a bug in the code, this is a annoyance when sequentially working through the test cases. A prior test case made the tester create the file and it thus now has to be overwritten